charts/mongodb/remedy.md

```diff
@@ -0,0 +1,90 @@
+## TODOs on MongoDB Critical Alerts
+
+### Database Alerts
+
+- #### MongoDBDown
+  - Describe the `MongoDB` CR, check the reason from conditions and try restarting the pods
+  - Check database pod logs
+  - Contact AppsCode team
+- #### MongodbReplicationLag
```

Comment:
All the `Mongodb` should be `MongoDB`

Reply:
The alerts are already named liked that

Reply:
some of them with `db`, & some others with `DB` ? That's strange !!